requirements.txt

```diff
@@ -31,6 +31,7 @@ flaskwebgui==0.3.7
 send2trash
 dependency_injector==4.40.0
 eventlet
+realesrgan
```

Comment:
@Vargol; Remind of why we're making this a hard requirement for *everyone*, regardless of whether they use esrgan or not?

Reply:
It's is the tests we do as part of the pre-flight testing, its part of embiggen,  preload-models.py doesn't fetch the GPFGAN models it its not installed.
Users will forget they didn't install it and then wonder why that prompt they copied doesn't work especially then they see that bit of dream.py saying its been initialised.

Reply:
I take your point(s), but the conda and pip installs are not equivalent (conda does weird stuff like specify "pytorch", which does not exist), so please *do not* base decisions about changes to either, given the state of the other.